src/lwc/geFormWidget/geFormWidget.js

```diff
@@ -59,4 +61,16 @@ export default class GeFormWidget extends LightningElement {
     get totalAmount() {
         return checkNestedProperty(this.widgetData, 'donationAmount') ? this.widgetData.donationAmount : 0;
     }
+
+    /**
+     * @description This method should return a list of DataImport__c field api names
+     *              that the widget uses so that parent components (geFormSection) know
+     *              which fields to include when calling this component's load() function.
+     * @returns An Array of field names used by this component.
+     */
+    @api
+    get allFieldsByAPIName() {
```

Comment:
Should this come from the geFormWidgetAllocation and be passed up here? Just thinking ahead to other widgets that aren't necessarily using this same field

Reply:
Yep!  Will move it